bears/vcs/git/GitCommitBear.py

```diff
@@ -353,3 +356,11 @@ def check_issue_reference(self, body,
                 elif not compiled_issue_no_regex.fullmatch(reference.group(1)):
                     yield Result(self, 'Invalid issue number: '
                                        '{}'.format(issue))
+
+    def check_revert_commit(self, shortlog, body):
+        """
+        Check for revert commits
+        and that they have a reason
```

Comment:
full stop expected. Also not clear what is meant by `and they have a reason`

Reply:
if you are using this code only once better not make a function for it.

Reply:
rather it sud be `which has a reason` ;P

Reply:
commits is plural so have should be correct 😜